dnf/base.py

```diff
@@ -923,51 +911,39 @@ def display_banner(pkg, count):
                 continue
             po = installed[0]
             count = display_banner(rpo, count)
-            yumdb_info = self._yumdb.get_package(po)
-            yumdb_info.from_repo = rpo.repoid
-
-            yumdb_info.reason = tsi._propagated_reason(self._yumdb, self._get_installonly_query())
-            yumdb_info.releasever = self.conf.releasever
-            if hasattr(self, 'args') and self.args:
-                yumdb_info.command_line = ' '.join(self.args)
-            elif hasattr(self, 'cmds') and self.cmds:
-                yumdb_info.command_line = ' '.join(self.cmds)
-            csum = rpo.returnIdSum()
-            if csum is not None:
-                yumdb_info.checksum_type = str(csum[0])
-                yumdb_info.checksum_data = csum[1]
```

Comment:
It should be - yumdb_info is package_data + trans_data + some package attributes in SWDB context. These objects are already being created elsewhere.